data_model/clusters/FanControl.xml

```diff
@@ -119,36 +119,36 @@ Davis, CA 95616, USA
       </item>
     </enum>
     <enum name="FanModeSequenceEnum">
-      <item value="0" name="Off" summary="Fan is capable of off, low, medium and high modes">
+      <item value="0" name="OffLowMediumHigh" summary="Fan is capable of off, low, medium and high modes">
```

Comment:
These files are to be modified by the XML data model parser. We should not update them manually.

Reply:
Reverted...